gnovm/stdlibs/context.go

```diff
@@ -18,3 +18,18 @@ type ExecContext struct {
 	OrigSendSpent *std.Coins // mutable
 	Banker        Banker
 }
+
+// XXX: migration from InjectStdlibs to NativeStore meant that stdlibs can
+// no longer directly access ExecContext, as it would create a circular
+// dependency; hence the helper functions, so access can still be done through
+// interface type assertions.
+// These probably need to have a better place.
```

Comment:
@thehowl is it still valid ? `ExecContext` is part of the `stdlibs` package, so where is the circular dependency ?

Reply:
if we allowed to use stdlibs.ExecContext, stdlibs/native.go -> calls native function, which does type assertion for stdlibs.ExecContext -> import stdlibs

i'll try out putting exec context in stdlibs/std, might be a good place

Reply:
Yes why not I agree this can be a good place.